### DISCUSSION TITLE
请增加编辑状态下的样式。

### DISCUSSION THREAD

**Discussion by taihangyishui** [dotnetcore/BootstrapBlazor#4797] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

 像这种<EditorItem @bind-Field="@context.DemandName" Readonly="true" />使用，如果Readonly这种字段使用过多时，会显的界面一片灰色。

### Describe the solution you'd like

建议增加个浅色选项，或在该种字段前方或末尾 加上“锁”图标。这样更清新。

**Comment by bb-auto** [comment]
@taihangyishui Thank you for contacting us. We will give feedback later.
